docs/sources/developers/http_api/folder.md

```diff
@@ -118,6 +122,11 @@ Content-Type: application/json
 }
 ```
 
+If nested folders are enabled, and the folder is nested (lives under another folder) then the response additionally contains:
+
+- **parentUid** - the parent folder UID.
+- **parents** - an array with the whole tree hierarchy starting from the root going down up to the parent folder.
```

Comment:
Should "down" and "up" both be in the description of **parents**? Or are we just missing a comma?
Whatever fix we make here should be applied to lines 198, 199, 269, and 270.

```suggestion
- **parentUid** - The parent folder UID.
- **parents** - An array with the whole tree hierarchy, starting from the root going down up to the parent folder.
```